tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Grpc/GrpcLegacy/Client/DefaultCallInvokerInstrumentation.cs

```diff
@@ -17,7 +17,7 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Grpc.GrpcLegacy.Client
         AssemblyName = "Grpc.Core",
         TypeName = "Grpc.Core.DefaultCallInvoker",
         MethodName = "CreateCall",
-        ReturnTypeName = "Grpc.Core.CallInvocationDetails`2<!!0,!!1>",
+        ReturnTypeName = "Grpc.Core.CallInvocationDetails`2[!!0,!!1]",
```

Comment:
Do we _actually_ use the `ReturnType` for anything? If not, is there any value in it here... obviously we've been specifying rubbish up to now, maybe we should just remove it from the attribute entirely? Seems little point in creating and sending that string to the native side if we're not doing anything with it?

On the other hand, it's helpful for "documentation" (but _is_ that documentation useful if it's wrong 😉)

Reply:
Yeah I'm not sure if we want to remove it or not... in case we do, definitively must be in another PR, the impact of the change is also big in the native side I think (structure changes, method signature comparison, etc).